<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SN_PointerUI</class>
 <widget class="QMainWindow" name="SN_PointerUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SAGENext Pointer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="hookMouseBtn">
        <property name="text">
         <string>share pointer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="isConnectedLabel">
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>242</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Connection"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Connection">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/powerbutton_off.med.png</normaloff>
     <normalon>:/images/powerbutton_off.med.png</normalon>:/images/powerbutton_off.med.png</iconset>
   </property>
   <property name="text">
    <string>New Connection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPointer_Name">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="actionShare_desktop">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/vnc_logo.svg.png</normaloff>
     <normalon>:/images/vnc_logo.svg.png</normalon>:/images/vnc_logo.svg.png</iconset>
   </property>
   <property name="text">
    <string>share desktop</string>
   </property>
   <property name="toolTip">
    <string>Share desktop</string>
   </property>
  </action>
  <action name="actionSend_text">
   <property name="icon">
    <iconset>
     <normalon>:/images/letter_T_48.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>send text</string>
   </property>
   <property name="toolTip">
    <string>Send text to current application</string>
   </property>
  </action>
  <action name="actionOpen_Media">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/opendir_128.png</normaloff>
     <normalon>:/images/opendir_128.png</normalon>:/images/opendir_128.png</iconset>
   </property>
   <property name="text">
    <string>Open Media</string>
   </property>
   <property name="toolTip">
    <string>Send media file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
